spec/form_models/jobseekers/job_application/review_form_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe Jobseekers::JobApplication::ReviewForm, type: :model do
+  let(:completed_steps) { [] }
+
+  subject do
+    described_class.new({ confirm_data_accurate: "1", confirm_data_usage: "1", completed_steps: completed_steps })
+  end
+
+  it { is_expected.to validate_acceptance_of(:confirm_data_accurate) }
+  it { is_expected.to validate_acceptance_of(:confirm_data_usage) }
+
+  context "when all steps are complete" do
+    let(:completed_steps) { JobApplication.completed_steps.keys }
+
+    it "is valid" do
+      expect(subject).to be_valid
+    end
+  end
+
+  context "when there are incomplete steps" do
+    let(:completed_steps) { JobApplication.completed_steps.keys.sample(2) }
```

Comment:
A little voice in my head is saying "tests should be deterministic" ...

Reply:
Good point